SpatialGDK/Source/SpatialGDK/Private/Interop/Connection/SpatialWorkerConnection.cpp

```diff
@@ -598,11 +598,25 @@ void USpatialWorkerConnection::ProcessOutgoingMessages()
 		{
 			FCreateEntityRequest* Message = static_cast<FCreateEntityRequest*>(OutgoingMessage.Get());
 
+#if TRACE_LIB_ACTIVE
```

Comment:
Nit: it seems we can have fewer lines of code duplicated. i.e. have just one call to `Worker_Connection_SendCreateEntityRequest`.

Reply:
Updated this